.github/workflows/docker_build.yml

```diff
@@ -84,17 +95,43 @@ jobs:
           username: ${{ secrets.REGISTRY_USERNAME }}
           password: ${{ secrets.REGISTRY_PASSWORD}}
 
-      - name: Build and push
+      - name: Set docker image tar name
+        id: setimgver
+        run: |
+          calculatedSha=$(git rev-parse --short ${{ github.sha }})
+          echo "::set-output name=IMAGE_TAR_FULLNAME::${{ inputs.project }}-${{ env.COMPONENT_NAME }}-$calculatedSha.tar"
+      
+      - name: Build
         uses: docker/build-push-action@v2
         with:
           context: ${{ inputs.docker_context }}
           file: ${{ inputs.docker_file }}
+          builder: ${{ steps.buildx.outputs.name }}
           push: ${{ inputs.push }}
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
+          # outputs: type=image,dest=/tmp/${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}
           build-args: |
             OMNI_COMPONENT=${{ env.COMPONENT_NAME }}
             NODE_ENV=${{ inputs.node_env }}
             NPM_TOKEN=${{ secrets.NPM_TOKEN }}
             OMNI_COMPONENT_COMMIT=${{ github.sha }}
-            OMNI_COMPONENT_VERSION=${{ steps.meta.outputs.version }}
+            OMNI_COMPONENT_VERSION=${{ steps.meta.outputs.version }}       
+      
+
+          
+      - name: save docker image to tar file
+        run: docker save --output ./${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }} ${{ steps.meta.outputs.tags }}
+
+      - name: upload docker image to private minio storage
+        run: |   
+          export AWS_ACCESS_KEY_ID=${{ secrets.MINIO_CN_ACCESS_KEY_ID }}
+          export AWS_SECRET_ACCESS_KEY=${{ secrets.MINIO_CN_SECRET_ACCESS_KEY }}
+          aws --endpoint-url ${{ secrets.MINIO_CN_ENDPOINT_URL }} s3 cp  ./${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }}  s3://docker-img-artifacts/${{ steps.setimgver.outputs.IMAGE_TAR_FULLNAME }} 
```

Comment:
we will only do this when the `MINIO_CN_ENDPOINT_URL` `MINIO_CN_SECRET_ACCESS_KEY` and `MINIO_CN_ACCESS_KEY_ID` are not empty